docs/architecture/storage/primitives.md

```diff
@@ -0,0 +1,98 @@
+# Primitives
+
+## TrieKey
+
+Describes all keys which may be inserted to storage:
+
+* `Account`
+* `ContractCode`
+* `AccessKey`
+* `ReceivedData`
+* `PostponedReceiptId`
+* `PendingDataCount`
+* `PostponedReceipt`
+* `DelayedReceiptIndices`
+* `DelayedReceipt`
+* `ContractData`
+* `YieldedPromiseQueueIndices`
+* `YieldedPromiseQueueEntries`
+* `PromiseYieldReceipt`
```

Comment:
Could you quickly add three more keys here to the list? `BufferedReceiptIndices`, `BufferedReceipt` and `BandwidthSchedulerState`